.github/workflows/deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Add AWS CLI
+      run: apt-get update && apt-get install -y awscli
```

Comment:
the distribution package is usually outdated, consider installing it as recommended in their doc :)